ibis/backends/conftest.py

```diff
@@ -307,8 +308,12 @@ def pytest_collection_modifyitems(session, config, items):
                     item,
                     pytest.mark.xfail(
                         (
-                            sys.version_info >= (3, 11)
-                            and not isinstance(item, pytest.DoctestItem)
+                            not isinstance(item, pytest.DoctestItem)
+                            and (
+                                sys.version_info >= (3, 11)
+                                or vparse(pd.__version__) >= vparse("2")
+                                or vparse(np.__version__) >= vparse("1.24")
```

Comment:
Shouldn't the latter be enough?

Reply:
No, this existed before this PR, I'm just doing it here to avoid having to make another PR and rebase this one.